src/mame/heathkit/h_88_cass.cpp

```diff
@@ -0,0 +1,166 @@
+// license:BSD-3-Clause
+// copyright-holders:Mark Garlanger
+/***************************************************************************
+
+  Heathkit H-17 Floppy controller
+
+    This was an option for both the Heathkit H8 and H89 computer systems.
+
+****************************************************************************/
+
+#include "emu.h"
+
+#include "h_88_cass.h"
+
+#include "machine/clock.h"
+
+#include "speaker.h"
+
+#define LOG_REG   (1U << 0)
+#define LOG_LINES (1U << 1)
+#define LOG_CASS  (1U << 2)
+#define LOG_FUNC  (1U << 3)
```

Comment:
You can’t use `(1U << 0)` for one of your own categories, as it’s used for the predefined `LOG_GENERAL`.